tests/features/project.feature

```diff
@@ -39,17 +62,48 @@ Feature: Project content creation
     And I fill in "Navigation title" with "My navigation title text" in the "Alternative titles and teaser" region
     And I fill in "Departments" with "Audit Board of the European Communities"
 
-    # Add link fields.
     When I press "Save"
     Then I should see "My Project"
     And I should not see "EU financing"
-    And I should not see "Committee on Agriculture and Rural Development"
     And I should see "European Patent Office"
-    And I should see "Audit Board of the European Communities"
-    And I should see "Anti Fraud Information System (AFIS)"
-    And I should not see "Project teaser"
-    And I should see "Summary text"
     And I should see "Body text"
+    # Coordinators
+    And I should see "Coordinators stakeholder"
+    And I should see "CS"
+    And I should see "France"
+    And I should see "Ma société 1"
+    And I should see "1 rue de la Paix"
+    And I should see "Etage 1"
+    And I should see "11111"
+    And I should see "Cruscades"
+    And I should see "1"
+    And I should see "https://ec.europa.eu/contact1"
+    Then I should see "Logo" in the "Project coordinators" region
+    And I should see "https://ec.europa.eu/1"
```

Comment:
@drishu sorry, I mean to add ` in the "Project coordinators" region` to all the assertions here above, same for the Participants

Reply:
Added regions to assertions